packages/editor/src/store/focus/selectors.ts

```diff
@@ -12,27 +10,3 @@ export const selectIsFocused = createSelector(
   [selectSelf, (_state, id: string) => id],
   (focus, id: string) => focus === id
 )
-
-export const selectHasFocusedChild = createSelector(
-  [(state: State) => state, (_state, id: string) => id],
-  (state, id: string) => {
-    const tree = selectChildTree(state, id)
-    if (!tree || !tree.children) return false
-    const focused = selectFocused(state)
-    return R.any((node) => node.id === focused, tree.children)
-  }
-)
-
-export const selectHasFocusedDescendant = createSelector(
```

Comment:
crazy that we only needed this for the placeholder, would have thought we use it for other behaviours as well.

Reply:
We didn't use it for placeholder either 😁

Reply:
ah okay :)